### ISSUE TITLE
Top1 instead of Top5 for chart.png

### ISSUE THREAD

**Issue by LukeAI** [AlexeyAB/darknet#4280]
Is it possible to see top1 instead of top5 in chart.png?
I'm training a classifier with a small number of classes so top5 is always 100% it'd be great to see how it is converging and if I am overfitting or not.

**Comment by AlexeyAB** [comment]
set `top=1` in the `obj.data` file

**Comment by LukeAI** [comment]
thanks for rapid reply!
unfortunately it hasn't worked.
I have 3 classes so top3 and top5 are always 100%
![chart](https://user-images.githubusercontent.com/43993778/68690003-38bf7d80-0569-11ea-9a9d-26c58c169f78.png)

**Comment by LukeAI** [comment]
semi-related: Is it possible to set how often topk validation accuracy is calculated? I do want to see the running val accuracy but at the moment I'm training a network for 100,000 batches and it stops to calculate validation every 100 batches or so - so it spends around half the time checking val. It would train substantially faster if it only checked every 1000 batches.

**Comment by AlexeyAB** [comment]
Change this line and recompile: https://github.com/AlexeyAB/darknet/blob/c516b6cb0a08f82023067a649d10238ff18cf1e1/src/classifier.c#L147

`calc_topk_for_each = 1000`

**Comment by LukeAI** [comment]
cool, thx have done so, will save some time.
